bindings/src/test/java/io/pravega/storage/hdfs/HDFSStorageTest.java

```diff
@@ -64,7 +65,7 @@
 
     @Before
     public void setUp() throws Exception {
-        this.baseDir = Files.createTempDirectory("test_hdfs").toFile().getAbsoluteFile();
+        this.baseDir = Files.createTempDirectory("test_hdfs_" + UUID.randomUUID().toString()).toFile().getAbsoluteFile();
```

Comment:
Why aren't we using junit TemporaryFolder rule?

Reply:
Done